File: WebApp/Scripts/app/model/TC_MMAGEN.js
Ext.define('WEBAPP.model.TC_MMAGEN', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'MMCODE', type: 'string' },
        { name: 'AGEN_NAMEC', type: 'string' },
        { name: 'PUR_UNIT', type: 'string' },
        { name: 'PUR_SEQ', type: 'string' },        
        { name: 'IN_PURPRICE', type: 'string' },
        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' }
    ]
});

File: WebApp/Models/TC/TC_MMAGEN.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class TC_MMAGEN : JCLib.Mvc.BaseModel
    {
        public string MMCODE { get; set; }
        public string MMNAME_C { get; set; }
        public string AGEN_NAMEC { get; set; }
        public string PUR_UNIT { get; set; }
        public double IN_PURPRICE { get; set; }
        public double PUR_SEQ { get; set; }        
        public DateTime CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public DateTime UPDATE_TIME { get; set; }
        public string UPDATE_IP { get; set; }
        public string UPDATE_USER { get; set; }

    }
}